modules/dmrpp_module/build_dmrpp_h4/build_dmrpp_h4.cc

```diff
@@ -51,18 +51,22 @@ using namespace build_dmrpp_util_h4;
 
 void usage() {
     const char *help = R"(
-    build_dmrpp -h: Show this help
+    build_dmrpp_h4 -h: Show this help
 
-    build_dmrpp -V: Show build versions for components that make up the program
+    build_dmrpp_h4 -V: Show build versions for components that make up the program
 
-    build_dmrpp -f <data file> -r <dmr file> [-u <href url>]: As above, but uses the DMR
-       read from the given file.
+    build_dmrpp_h4 -f <data file> -r <dmr file> [-u <href url>] [-M] [-D] [-v] [-V] [-d] 
 
-    Other options:
-        -v: Verbose
-        -d: Turn on BES software debugging output
-        -M: Add information about the build_dmrpp software, incl versions, to the built DMR++
-        -D: Disable the generation of HDF-EOS2/HDF4 missing latitude/longitude)";
+    options:
+        -f: HDF4/HDF-EOS2 file to build DMR++ from
+        -r: DMR file to build DMR++ from
+        -u: The href value to use in the DMR++ for the data file
+        -M: Add information about this software, incl versions, to the built DMR++
+        -D: Disable the generation of HDF-EOS2/HDF4 missing latitude/longitude
+        -h: Show this help
+        -v: Verbose information helpful for debugging and understanding the program working flow
+        -V: Show build versions for components that make up the program
+        -d: Turn on BES software debugging output)";
```

Comment:
There is an extra ')' at the end of the text on this line.

Reply:
That ) has to be there for the raw string literal to work.